api/src/main/java/io/opencensus/trace/propagation/BinaryFormat.java

```diff
@@ -65,23 +65,58 @@
   static final NoopBinaryFormat NOOP_BINARY_FORMAT = new NoopBinaryFormat();
 
   /**
-   * Serializes a {@link SpanContext} using the binary format.
-   *
+   * @deprecated use {@link #toByteArray(SpanContext)}.
    * @param spanContext the {@code SpanContext} to serialize.
    * @return the serialized binary value.
    * @throws NullPointerException if the {@code spanContext} is {@code null}.
    */
-  public abstract byte[] toBinaryValue(SpanContext spanContext);
+  @Deprecated
+  public byte[] toBinaryValue(SpanContext spanContext) {
+    return toByteArray(spanContext);
+  }
 
   /**
-   * Parses the {@link SpanContext} from the binary format.
+   * Serializes a {@link SpanContext} using the binary format into a byte array.
    *
+   * @param spanContext the {@code SpanContext} to serialize.
+   * @return the serialized binary value.
+   * @throws NullPointerException if the {@code spanContext} is {@code null}.
+   */
+  public byte[] toByteArray(SpanContext spanContext) {
```

Comment:
Why not make this abstract? (and fromByteArray)

Reply:
Because if in tests let's say somebody uses the previous API they override the toBinaryValue and if I make this abstract I will break their build.